### ISSUE TITLE
Third-party authentification changes

### ISSUE THREAD

**Issue by DirtyF** [sudweb/talks#15]
When running `npm run start` and calling the Google Spreadsheet key locally the following message is displayed:

> Access to Google Sheets data from this web page will soon start failing.  For information on fixing this, see https://g.co/sheets/3p-auth-changes

> … we will begin requiring explicit authorization when third-party sites request access to Google Sheets content via the Google Visualization API or Google Query Language.

> For more details on the technical changes required, please visit the Google Charts API Documentation. We will begin enforcing these requirements on **September 14, 2016**.$

/cc @oncletom

**Comment by oncletom** [comment]
> Note: Spreadsheets shared to "anyone who has the link can view" do not require credentials. Changing your spreadsheet's sharing settings is much easier than implementing authorization.

Cf. https://developers.google.com/chart/interactive/docs/spreadsheets#authorization

It will continue to work as is, as long as the CFP spreadsheets are shared as *anyone who has the link can view* (like it has been done until now).

**Comment by DirtyF** [comment]
Very fine, it won't much be used locally anyway. Thanks @oncletom, sorry for bothering.

**Comment by oncletom** [comment]
Locally or not, you will have the same issue while generating the website via Travis CI.

**Comment by DirtyF** [comment]
@oncletom I checked online, and the warning also appears. It's no big deal if the app still do the job.

Honestly I did try to follow your tracks along the way but I don't write JS apps like you do on a daily basis. I'm doin' my best here, playin' with all these tools I barely know. You did a terrific job on your own and now we got to pay the price for relying on a SPOF and not having managed to have two different persons working on this. It's what happened and we weal deal with this.
